src/conversions.cc

```diff
@@ -34,7 +34,7 @@ void InitConversions(Local<Object> exports) {
     v8::Local<v8::Object> bufferView;
     bufferView = node::Buffer::New(Isolate::GetCurrent(), point_transfer_buffer, 0, 2 * sizeof(uint32_t)).ToLocalChecked();
     auto js_point_transfer_buffer = node::Buffer::Data(bufferView);
-  #elif V8_MAJOR_VERSION >= 8
+  #elif (V8_MAJOR_VERSION > 8 || (V8_MAJOR_VERSION == 8 && V8_MINOR_VERION > 3))
```

Comment:
Introduced in 8.3.9 https://github.com/v8/v8/commit/5cf02f0f207c05cebce74b1a8f2d0fe2accc7df6